### ISSUE TITLE
Download Issues

### ISSUE THREAD

**Issue by bnb** [nodejs/build#2557]
I've been seeing a number of people asserting that they can't download Node.js.

This has been relatively consistent on Twitter (compared to literally zero reports of this normally) over the past week or two, usually specifying that it's the MSI that they can't download.

I can't personally reproduce it, but I figured I should probably make an issue regardless since it seems to not be isolated.

**Comment by targos** [comment]
Can you share an example of those Twitter reports?

**Comment by mhdawson** [comment]
Downloads working for me as well. We'll need more specific info. Maybe it's geography related?

**Comment by bnb** [comment]
[This](https://twitter.com/Tony30B/status/1366444808160567299) is the first one I saw. I figured it was just user error. I saw another one in the day or two after that and then a third the day I posted this, which prompted me to create this issue. I'm struggling to find the others with keyword searches, now. Normally we get _zero_ reports of this to the Twitter handle, so three within a few days is... abnormal.
